configserver/src/main/java/com/yahoo/vespa/config/server/metrics/ProtonMetricsAggregator.java

```diff
@@ -0,0 +1,99 @@
+package com.yahoo.vespa.config.server.metrics;
+
+import com.yahoo.slime.Inspector;
+import java.util.List;
+
+public class ProtonMetricsAggregator {
```

Comment:
Two notes on this class:

1. An application can have more than one content cluster, so it'd be useful if these metrics are aggregated per cluster.
2. If I'm reading this right, the aggregation just sums the values. At least the percentage based metrics should be averaged instead

Reply:
Fixed in aggregator + cluster retriever.